### DISCUSSION TITLE
Add string data type in score config

### DISCUSSION THREAD

**Discussion by emfhal** [langfuse/langfuse#2402] [Ideas]
### Describe the feature or potential improvement

Add a 'string' data type to the score config. This would allow you to include comments as scores, such as 'right answer' and more. Currently, you can only choose numbers or categories.

**Comment by marliessophie** [comment]
Hi @emfhal, thanks for the feedback! Extending the data types is definitely something we are considering. If you are looking to add comments though, leaving two thoughts: 
- You can already add comments to individual scores, if you click on the comment icon next to a score in the annotation drawer (see screenshot below). Maybe that helps? 
- We're planning a wider project in the near feature that will allow you to add comments on objects across langfuse such as traces, observations, sessions etc. In this case we'd expect comments to apply to the full object, not just individual scores. 
If you have any feedback/questions please let us know.
<img width="1267" alt="Screenshot 2024-06-27 at 09 02 06" src="https://github.com/langfuse/langfuse/assets/74332854/ae0ad6d9-42d7-4692-9c3b-0d936c4e21e5">
